### DISCUSSION TITLE
v1.4 Roadmap - HA NFS - RWX

### DISCUSSION THREAD

**Discussion by voarsh2** [longhorn/longhorn#4016] [Q&A]
I saw in the roadmap that there's a possibility of having RWX more highly available than is currently - but I don't see an active issue yet?

**Comment by PhanLe1010** [comment]
@innobead Do you know the ticket? I couldn't find it neither

**Reply by innobead** [reply]
@voarsh2 https://github.com/longhorn/longhorn/issues/2293

cc @derekbit
